lib/ai_bot/personas/persona.rb

```diff
@@ -10,7 +10,8 @@ def vision_enabled
           end
 
           def vision_max_pixels
-            1024 * 1024
+            # 1024 * 1024
```

Comment:
Seems unnecessary to keep this comment here. We _could_ do `1.megabyte` but it's not clear how that translates to pixels. Would be cool if rails could do `1.megapixel`

Reply:
removed